src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/EventMessageReceiverServiceTest.java

```diff
@@ -232,6 +244,118 @@ void test_handle_dql_message_valid_message() throws JsonProcessingException {
 
         verify(caseEventMessageRepository).save(caseEventMessageEntityCaptor.capture());
         assertEquals(MessageState.NEW, caseEventMessageEntityCaptor.getValue().getState());
+        assertTrue(caseEventMessageEntityCaptor.getValue().getFromDlq());
+    }
+
+    @Test
+    void test_handle_ccd_case_event_asb_message_feature_flag_disabled() throws JsonProcessingException {
```

Comment:
1. can we rename this to should_handle_ccd_case_event_asb_message_when_feature_flag_disabled()?
2. Can we rename all the tests to follow this approach?

Reply:
Fixed in https://github.com/hmcts/wa-case-event-handler/pull/201/commits/1181a613c288a1a9a1bffa020681c41a4824a9e5